internal/k6runner/k6runner.go

```diff
@@ -22,9 +22,24 @@ import (
 	"github.com/spf13/afero"
 )
 
+// Script is a k6 script that a runner is able to run, with some added instructions for that runner to act on.
+type Script struct {
+	Script   []byte   `json:"script"`
+	Settings Settings `json:"settings"`
+	// TODO: Add Metadata and Features.
```

Comment:
Here is where, in the future, we will add metadata (e.g. check ID, tenant ID) for the runners to log later, and script features (e.g. browser enabled, k6 version).